make/kube

```diff
@@ -6,24 +6,24 @@ GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 cd "${GIT_ROOT}"
 source .envrc
 
-CREATE_HELM_CHART=false
-BUILD_TARGET=kube
-
 if [ "${1:-}" = "helm" ]; then
-    CREATE_HELM_CHART=true
     BUILD_TARGET=helm
-fi
-
-if [ "${BUILD_TARGET}" = "kube" ]; then
+else
+    BUILD_TARGET=kube
     # Overrides when generating kube config files instead of helm charts.
     FISSILE_OUTPUT_DIR="${PWD}/kube"
     ENV_FILES="$(echo bin/settings/*.env bin/settings/kube/*.env)"
+    DOMAIN=${DOMAIN:-$(grep '^DOMAIN=' bin/settings/network.env | sed 's/DOMAIN=//')}
```

Comment:
Agreed. Missed that. The `run` script does not have however. I would propose to go with how Jan is doing it, i.e. add `cd ${GIT_ROOT}` at the beginning and then remove all the other `GIT_ROOT` refs.